vita3k/modules/module_parent.cpp

```diff
@@ -157,15 +157,21 @@ void call_import(EmuEnvState &emuenv, CPUState &cpu, uint32_t nid, SceUID thread
 bool load_module(EmuEnvState &emuenv, SceUID thread_id, SceSysmoduleModuleId module_id) {
     LOG_INFO("Loading module ID: {}", log_hex(module_id));
 
-    const auto module_paths = sysmodule_paths[module_id];
+    const auto &module_paths = sysmodule_paths[module_id];
 
     for (std::string module_path : module_paths) {
-        module_path = "sys/external/" + module_path + ".suprx";
-
+        auto device = VitaIoDevice::vs0;
+        // todo: move loading SCE_SYSMODULE_ULT from module preload to here
+        if (module_id == SCE_SYSMODULE_SMART || module_id == SCE_SYSMODULE_FACE) {
+            module_path = "sce_module/" + module_path + ".suprx";
+            device = VitaIoDevice::app0;
+        } else {
+            module_path = "sys/external/" + module_path + ".suprx";
+        }
```

Comment:
Set libult here just broken all 👌
Revert this all code is wrong, i have said you, you need set load this module inside pré load in interface